### PR TITLE
feature/chris-7434-remove-rx-tabs

### PR DESCRIPTION
**Pull request by alexandec**
## Description of Change

Remove the tabs and reorder items on the Prescription History Screen. This is the first of two PRs, with the second ticket being #7355 which will update the filtering/sorting. We are using a separate branch so all the changes will merge to develop together.

## Screenshots/Video

Before/after:
&nbsp;&nbsp;

## Testing

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations

- Remove tabs of All, Pending, and Tracking under the Start refill request
- Move "We can't refill...in the app" to below Start refill request button

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.test.tsx [hunk 1]
```diff
@@ -256,15 +256,11 @@ const prescriptionData: PrescriptionsGetData = {
...
-      undefined,
-      {
-        setParams: jest.fn(),
-        setOptions: jest.fn(),
-      },
-      { params: { startingTab } },
-    )
+  const initializeTestInstance = (includeTransferred = false) => {
+    const props = mockNavProps(undefined, {
+      setParams: jest.fn(),
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.test.tsx:261]: I don't think we need the setParams anymore do we?

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx [hunk 2]
```diff
@@ -515,35 +461,8 @@ const PrescriptionHistory: FC<PrescriptionHistoryProps> = ({ navigation, route }
...
-    borderBottomWidth: 1,
-    borderColor: 'primary',
-  }
-
   const hasNoItems = prescriptions?.length === 0
 
-  const getInstructions = () => {
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return t('prescriptions.header.helper.all')
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx:528]: can't these strings be removed from the common translation file?

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx [hunk 3]
```diff
@@ -515,35 +461,8 @@ const PrescriptionHistory: FC<PrescriptionHistoryProps> = ({ navigation, route }
...
-        return t('prescriptions.header.helper.pending')
-      case PrescriptionHistoryTabConstants.TRACKING:
-        return t('prescriptions.header.helper.tracking')
-    }
-  }
-
-  const getInstructionA11y = () => {
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return a11yLabelVA(t('prescriptions.header.helper.all'))
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx:539]: same thing here with the strings

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx [hunk 4]
```diff
@@ -601,32 +516,30 @@ const PrescriptionHistory: FC<PrescriptionHistoryProps> = ({ navigation, route }
     )
   }
 
-  const getHistoryListHeader = () => {
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return t('prescription.history.list.title.all', { count: prescriptions?.length })
-      case PrescriptionHistoryTabConstants.PENDING:
-        return t('prescription.history.list.title.pending', { count: prescriptions?.length })
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx:609]: same thing here with the strings

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx [hunk 5]
```diff
@@ -3,45 +3,28 @@ import React, { FC } from 'react'
...
-const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ currentTab, isFiltered }) => {
+const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ isFiltered }) => {
   const { t } = useTranslation(NAMESPACE.COMMON)
   const theme = useTheme()
 
-  const getFilteredTabString = () => {
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return t('prescription.history.empty.filtered.all')
-      case PrescriptionHistoryTabConstants.PENDING:
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx:26]: same thing with the strings here

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx [hunk 6]
```diff
@@ -3,45 +3,28 @@ import React, { FC } from 'react'
...
-
   const getFilteredNoMatch = () => {
-    let contenta11y = {}
-    if (currentTab === PrescriptionHistoryTabConstants.ALL) {
-      contenta11y = { accessibilityLabel: a11yLabelVA(t('prescription.history.empty.filtered.all')) }
-    }
+    const contenta11y = { accessibilityLabel: a11yLabelVA(t('prescription.history.empty.filtered.all')) }
     return (
       <>
         <TextView textAlign={'center'} variant="MobileBodyBold">
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx:23]: I know this wasn't previously, but shouldn't this be a header?

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx [hunk 7]
```diff
@@ -52,56 +35,16 @@ const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ c
...
-          </>
-        )
-      case PrescriptionHistoryTabConstants.PENDING:
-        return (
-          <>
-            <TextView textAlign={'center'} variant="MobileBodyBold">
-              {t('prescription.history.empty.pending.title')}
-            </TextView>
-            <TextView
-              pt={theme.dimensions.condensedMarginBetween}
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx:78]: same thing with strings, can't some be removed from common translation file?

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx [hunk 8]
```diff
@@ -52,56 +35,16 @@ const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ c
...
-              variant="MobileBody"
-              accessibilityLabel={a11yLabelVA(t('prescription.history.empty.tracking.p2'))}>
-              {t('prescription.history.empty.tracking.p2')}
-            </TextView>
-          </>
-        )
-    }
+    return (
+      <>
+        <TextView textAlign={'center'} variant="MobileBodyBold">
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx:40]: shouldn't this also be a header, I know it wasn't previously

**Review by Sparowhawk — CHANGES_REQUESTED** [review]

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx (hunk 6 above)
Comment (alexandec) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx]: Make sense to me, added the accessibilityRole

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx (hunk 8 above)
Comment (alexandec) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx]: Make sense to me, added the accessibilityRole

File: VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx (hunk 7 above)
Comment (alexandec) [reply VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx:78]: Removed prescription.history.empty.pending.* and prescription.history.empty.tracking.*

**Pull request merged**